### DISCUSSION TITLE
SYSTEM > SENSORS problem

### DISCUSSION THREAD

**Discussion by kocjan1** [edterbak/HeishaMoNR#166] [Q&A]
Hi

I configured SENSORS tab with outside temp for WAR, inside temp for RTC, power production and consumption but it disappears after few days and i need to set up them again. Also some settings are reset like RTC, SoftStart, WAR and i need to them ON again. What i do wrong? Why this setting clears?

**Comment by geduxas** [comment]
Are you running with HomeAssistant?

**Reply by kocjan1** [reply]
Yes, Node-RED addon 17.0.10

**Reply by geduxas** [reply]
So it's simple, I think you forgot fallow user guide for home assistant, and missing some setup steps.. so that's why after HA restart all data lost. 
Please falow this guide:
https://github.com/edterbak/NodeRed_Heishamon_control/tree/main/HomeAssistant

And if it's resolves your problem, don't forget to close issue.

**Reply by kocjan1** [reply]
I read the instructions. But in my version of HA settings.js file is in directory addon_configs/xxxxxxxx_nodered and if i add "contextStorage" setting nodered doesnt start. it crash with error - Error loading settings file: /etc/node-red/config.js
/config/settings.js:196
	default: "memoryOnly",
SyntaxError: Unexpected token 'default'

**Reply by kocjan1** [reply]
Ok, i found error. thanks.
